screens/SignIn.js

```diff
@@ -1,79 +1,73 @@
 import React from 'react';
 import {
-  StyleSheet, TextInput, Button, View, Text
+  SafeAreaView,
+  Button,
+  ActivityIndicator,
 } from 'react-native';
 import { Formik } from 'formik';
+import * as yup from 'yup';
 import { retrieveSignInFunction } from '../services/parse/auth';
+import FormInput from '../components/FormInput';
 
-// props should be passed in here if needed
-export default function SignIn() {
+const validationSchema = yup.object().shape({
+  username: yup
+    .string()
+    .label('Username')
+    .required(),
+  password: yup
+    .string()
+    .label('Password')
+    .required()
+    .min(7, 'Seems a bit short...')
```

Comment:
yo! makes this 4 :D

Reply:
we have users who make their passwords 4 and it's annoying af, but something we have to make sure we're being inclusive

Reply:
cool. adjusted